huxley/www/static/js/huxley/components/CountrySelect.js

```diff
@@ -11,14 +16,8 @@ var CountrySelect = React.createClass({
   propTypes: {
     onChange: React.PropTypes.func,
     countries: React.PropTypes.array,
-    selectedCountryID: React.PropTypes.number
-  },
-
-  shouldComponentUpdate: function(nextProps, nextState) {
```

Comment:
Why are we removing this method? It's an important optimization to prevent us from rendering 1920-ish (192 x 10) React elements with every single keystroke.